pkg/provider/azure_vmss_cache.go

```diff
@@ -294,24 +307,36 @@ func (ss *ScaleSet) deleteCacheForNode(nodeName string) error {
 	return nil
 }
 
-func (ss *ScaleSet) newAvailabilitySetNodesCache() (*azcache.TimedCache, error) {
+func (ss *ScaleSet) newNonVmssUniformNodesCache() (*azcache.TimedCache, error) {
```

Comment:
Could you separate VMSSFlex caches from VMAS/Standalone VM caches for more clear?

Reply:
nonVmssUniformNodesCache is only used for checking node type here